### PR TITLE
Add PtBr 325 - 334

### PR DESCRIPTION
**Pull request by maisondasilva**

### CODE REVIEW

File: res/loc/po/pt-BR.po [hunk 1]
```diff
@@ -1807,3 +1807,43 @@ msgstr "Instalação estendida do Windows 11 (sem TPM / sem Secure Boot)"
...
+msgid "Customize Windows installation?"
+msgstr "Personalizar a instalação do Windows?"
+
+#. • MSG_329
+msgid "Remove requirement for 4GB+ RAM, Secure Boot and TPM 2.0"
+msgstr "Remova a exigência de 4 GB ou mais de RAM, inicialização segura e TPM 2.0"
+
+#. • MSG_330
+msgid "Remove requirement for an online Microsoft account"
+msgstr "Remover o requisito para uma conta online da Microsoft"
```
Comment (marcosfrm) [inline res/loc/po/pt-BR.po:1833]: Eu repetiria "exigência", mas você decide. Como são caixas de seleção sem uma pergunta, me parece que fica melhor "Remova" como na string anterior.

File: res/loc/po/pt-BR.po [hunk 2]
```diff
@@ -1807,3 +1807,43 @@ msgstr "Instalação estendida do Windows 11 (sem TPM / sem Secure Boot)"
...
+msgid "Prevent Windows To Go from accessing internal disks"
+msgstr "Impedir que o Windows To Go acesse discos internos"
+
+#. • MSG_333
+msgid "Create a local account with username:"
+msgstr "Crie uma conta local com nome de usuário:"
+
+#. • MSG_334
+msgid "Set regional options to the same values as this user's"
+msgstr "Defina as opções local com os mesmos valores deste usuário"
```
Comment (marcosfrm) [inline res/loc/po/pt-BR.po:1849]: "opções local" está errado: use "opções regionais".

File: res/loc/po/pt-BR.po [hunk 3]
```diff
@@ -1807,3 +1807,43 @@ msgstr "Instalação estendida do Windows 11 (sem TPM / sem Secure Boot)"
...
+msgid "Remove requirement for an online Microsoft account"
+msgstr "Remover o requisito para uma conta online da Microsoft"
+
+#. • MSG_331
+msgid "Disable data collection (Skip privacy questions)"
+msgstr "Desative a coleta de dados (Pular as perguntas sobre privacidade)"
+
+#. • MSG_332
+msgid "Prevent Windows To Go from accessing internal disks"
+msgstr "Impedir que o Windows To Go acesse discos internos"
```
Comment (marcosfrm) [inline res/loc/po/pt-BR.po:1841]: Da mesma forma, por consistência: "Impeça".

File: res/loc/po/pt-BR.po (hunk 1 above)
Comment (maisondasilva) [reply res/loc/po/pt-BR.po]: Alterado obrigado pela sugestão!

File: res/loc/po/pt-BR.po (hunk 2 above)
Comment (maisondasilva) [reply res/loc/po/pt-BR.po]: Alterado obrigado pela sugestão!

File: res/loc/po/pt-BR.po (hunk 3 above)
Comment (maisondasilva) [reply res/loc/po/pt-BR.po]: Alterado obrigado pela sugestão!

**Comment by pbatard** [comment]
Okay, since the existing Brazilian Portuguese translator has not manifested themselves, and if you are still willing to contribute an update to the upcoming version of Rufus, can I please ask you to take a look at the [latest changes](https://github.com/pbatard/rufus/blob/master/res/loc/ChangeLog.txt#L8-L54) and make sure that you provide me with an up to date translation?

Please be __very mindful__ that some messages may have changed since December (see https://github.com/pbatard/rufus/commits/master/res/loc/ChangeLog.txt).

Also, in order to help you further, here is part of the mail I sent to existing translators a couple weeks ago:

> In short, there are about 35 new/updated messages since the last translation round, and the way you should proceed translating these new messages is (after first making sure that you have Poedit installed from https://poedit.net/download) to simply download and run https://files.akeo.ie/pollock/pollock-1.5.exe and follow its directions.
>
> This will generate the base translation file that can then be automatically opened in Poedit, and you will find all the messages that need an update highlighted in orange.
>
> Please make sure to also read the "Notes for translators" in the lower right corner while editing a message, as it may contain some relevant information about the message to translate.
>
> Once you are done with Poedit, you should save the file so that pollock.exe can detect the change and if you want, before sending the update through email, you can actually to test your changes by simply running rufus-3.22_ALPHA.exe which will have been downloaded into the same directory, as it will use the updated rufus.loc file generated by pollock.
>
> With regards to testing and some explanation of what the messages are about:
>
> - MSG_144 and MSG_199 are for the ISO download feature, and you are not expected to be able to test them at this stage (unless you are trying to run Rufus on Windows 7, which you shouldn't since version 3.22 will be the last ever release of Rufus that can run on Windows 7).
>
> - MSG_294, which relates directly to Rufus 3.23 dropping support for Windows 7, can be tested by opening an elevated prompt and issuing the command:
>    rufus-3.22_ALPHA.exe -z61
>
> - MSG_323 is not expected to be tested (it should only happen when trying to open a corrupted image)
>
> - MSG_325 to MSG_335 can be tested by:
>    o Downloading the latest Windows 11 ISO (which you can do through the ISO download feature of Rufus: https://raw.githubusercontent.com/wiki/pbatard/rufus/images/download.gif
>    o Opening that ISO in Rufus and pressing START
>    Note that MSG_332 will only appear if selecting 'Windows To Go' under 'Image options'
>    These messages are basically used to customise the Windows installation, and remove the many annoyances Microsoft added in recent versions of Windows, that make installation of Windows a lot more painful than it should be, such as requiring an online Microsoft account, having to manually select "no" to all the privacy questions, etc.
>
> - MSG_336 can be tested with <Ctrl>-<P>. Basically, when enabled, it ensures that the log that is saved by Rufus in %LOCALAPPDATA%\Rufus is never erased but only added to (in short, it will keep the log for each session you run with Rufus forever).
>
> - MSG_900 to MSG_922 are actually not messages that are being used by Rufus at all (as a matter of fact these are not included when compiling the application) but instead they are meant to appear in the Microsoft Store listing for your locale at https://apps.microsoft.com/store/detail/rufus/9PC3H3V7Q9CH.
> Basically, you will find that there is a https://apps.microsoft.com/store/detail/rufus/9PC3H3V7Q9CH?hl=<LOCALE_ID> page for each of your language, with <LOCALE_ID> being one of:
>
> en-us, ar-sa, bg-bg, zh-cn, zh-tw, hr-hr, cs-cz, da-dk, nl-nl, fi-fi, fr-fr, de-de, el-gr, he-il, hu-hu, id-id, it-it, ja-jp, ko-kr, lv-lv, lt-lt, ms-my, nb-no, fa-ir, pl-pl, pt-br, pt-pt, ro-ro, ru-ru, sr-latn-rs, sk-sk, sl-si, es-es, sv-se, th-th, tr-tr, uk-ua, vi-vn.
>
> Obviously, you should look at how the https://apps.microsoft.com/store/detail/rufus/9PC3H3V7Q9CH?hl=en-us page is presented to get an idea of what I would like each localized page to look like, noting that MSG_910 and higher are simply listing the features that Rufus can provide.
>
> I will also make a note that MSG_905 ("Boot") is intended to be used as a term that users can search for in the Microsoft Store (as in "boot + USB" or something like that), and that I also invite you, for MSG_904 (which curiously does not appear explicitly in the Store listing... so much for Microsoft being open about the licensing of the software they provide) to have a look to see whether a https://www.gnu.org/licenses/gpl-3.0.<COUNTRY_ID>.html exists for your language and use that if it does (if not, just use https://www.gnu.org/licenses/gpl-3.0.html). For instance, you should find that https://www.gnu.org/licenses/gpl-3.0.ja.html or https://www.gnu.org/licenses/gpl-3.0.de.html or https://www.gnu.org/licenses/gpl-3.0.zh-cn.html or https://www.gnu.org/licenses/gpl-3.0.pt-br.html exist (though, strangely, I was not able to locate a translated version of the GPL for Spanish, which is strange).
>
> If you have any question or need some clarification regarding the use or the meaning of any of the new messages, please don't hesitate to ask as I will do my best to help you.

**Comment by maisondasilva** [comment]
@marcosfrm na linha MSG_905 não seria bom deixar boot? porque acho meio difícil a pessoa buscar por Inicializável na Microsoft Store! Meu ponto de vista!

**Comment by marcosfrm** [comment]
Talvez "Inicialização" fique melhor. Todas as principais traduções estão traduzindo essa string. Acho que devemos seguir...

**Comment by marcosfrm** [comment]
Done. I will keep "Inicialização" as the translation for MSG_905. Let's close this PR.

**Pull request closed without merging**